Makefile

```diff
@@ -1,35 +1,29 @@
-.PHONY: all-checks
+.PHONY: all-checks check coverage docs format package requirements test
```

Comment:
Please revert these changes; they're not related to the goal of deprecating 3.8.

Reply:
Are they OK for another PR or not at all?

Reply:
Is there some benefit? My sense is no - it's fine to do it either way.